sky/provision/gcp/constants.py

```diff
@@ -41,6 +41,13 @@
 # with ServiceAccounts.
 
 SKYPILOT_VPC_NAME = 'skypilot-vpc'
+SKYPILOT_GPU_DIRECT_VPC_PREFIX = 'skypilot-gpu-direct-'
+SKYPILOT_GPU_DIRECT_VPC_NUM = 5
+SKYPILOT_GPU_DIRECT_VPC_CIDR_PREFIX = '10.129'
+GPU_DIRECT_TCPX_INSTANCE_TYPES = [
```

Comment:
Should we allow `a3-ultragpu-8g` and `a4-highgpu-8g`? https://cloud.google.com/compute/docs/gpus

Reply:
Refer to the doc here https://cloud.google.com/kubernetes-engine/docs/how-to/gpu-bandwidth-gpudirect-tcpx#required-features-capabilities, `a3-ultragpu-8g` and a4 machine types support `GPUDirect RDMA`, which is different from `GPUDirect TCPX`, so I think we should not include them here.